### PR TITLE
fix(adminEmail): added a **dev** check to see wether we are currently…

### PR DESCRIPTION
**Pull request by joanofdart**
… in dev and if not, we are setting the email to 'developers@cascade8.com'

Related to #3206

### CODE REVIEW

File: env/env.prod.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
 export const production = true;
+export const dev = false;
```
Comment (Mussuranae) [inline env/env.prod.ts:2]: So for prod, it will be also developers@cascade8.com ?

File: apps/backend-functions/src/environments/environment.ts [hunk 2]
```diff
@@ -41,7 +41,9 @@ export const algolia = {
   adminKey: mockConfigIfNeeded('algolia', 'api_key')
 };
 
-export const adminEmail = mockConfigIfNeeded('admin', 'email');
+
+
+export const adminEmail = dev ? mockConfigIfNeeded('admin', 'password'): 'developers@cascade8.com';
```
Comment (GrandSchtroumpf) [inline apps/backend-functions/src/environments/environment.ts:46]: isn't it the opposite ?

Comment (joanofdart) [reply apps/backend-functions/src/environments/environment.ts:46]: local => **mockConfigIfNeeded('admin', 'password');**
staging => **'developers@cascade8.com'**
prod => environment.prod.ts => **functions.config().admin.email;**

File: env/env.prod.ts (hunk 1 above)
Comment (joanofdart) [reply env/env.prod.ts:2]: In prod we have the email assigned directly, e.g:
- **adminEmail = functions.config().admin.email;**

**Pull request merged**